src/proof/arith_proof.cpp

```diff
@@ -1068,90 +1130,91 @@ void LFSCArithProof::printTheoryLemmaProof(std::vector<Expr>& lemma,
         }
         default: Unreachable();
       }
-    }
+      }
```

Comment:
Something went wrong with the formatting here. This parentheses was moved even though it shouldn't have been. I think all the changes below are actually just due to this formatting change.

Reply:
Will touch this line and then clang format changes.